### PR TITLE
Update ru.m3u

### PR DESCRIPTION
**Pull request by freearhey**

### CODE REVIEW

File: streams/ru.m3u [hunk 1]
```diff
@@ -248,3 +248,77 @@ http://stream0.tv41.ru/live.m3u8
 https://live-echotv.cdnvideo.ru/echotv/echotv.sdp/playlist.m3u8
 #EXTINF:-1 tvg-id="Yuvelirochka.ru",Ювелирочка ТВ (576p)
 https://live-uvelirochka.cdnvideo.ru/uvelirochka/uvelirochka_720p3/playlist.m3u8
+#EXTINF:-1 tvg-id="",TVOE TV HD
+http://tvoetv.space:8080/tvoetv/index.m3u8
+#EXTINF:-1 tvg-id="",Кино 1 International
+http://kino-1.catcast.tv/content/38617/index.m3u8
```
Comment (Dum4G) [inline streams/ru.m3u:254]: We have streams/ru_catcast.m3u for that

File: streams/ru.m3u [hunk 2]
```diff
@@ -248,3 +248,77 @@ http://stream0.tv41.ru/live.m3u8
...
+#EXTINF:-1 tvg-id="",КИНО-1 INTERNATIONAL
+https://autopilot.catcast.tv/content/38617/index.m3u8
```
Comment (Dum4G) [inline streams/ru.m3u:256]: We have streams/ru_catcast.m3u for that

File: streams/ru.m3u [hunk 3]
```diff
@@ -248,3 +248,77 @@ http://stream0.tv41.ru/live.m3u8
...
+#EXTINF:-1 tvg-id="",Кино 1 International HD
+http://213.172.73.132:8081/nimble_live/kino_1_int/playlist.m3u8
+#EXTINF:-1 tvg-id="",ViP COMEDY HD
+http://dmi3y-tv.ru/hls/okko/CH_ViPCOMEDYHD.m3u8
```
Comment (Dum4G) [inline streams/ru.m3u:260]: We have that, it's nothing more than a mirror
https://github.com/iptv-org/iptv/blob/f37fffc95ddad37a0c4d7e6e73fab3b3d6b6ccc7/streams/ru_okkotv.m3u#L63C1-L63C59

File: streams/ru.m3u [hunk 4]
```diff
@@ -248,3 +248,77 @@ http://stream0.tv41.ru/live.m3u8
...
+#EXTINF:-1 tvg-id="",ИЛЛЮЗИОН+
+http://91.220.135.165:81/51/index.m3u8
+#EXTINF:-1 tvg-id="",Кино ТВ
+http://91.220.135.165:81/55/index.m3u8
+#EXTINF:-1 tvg-id="",Еврокино
+http://91.220.135.165:81/30/index.m3u8
+#EXTINF:-1 tvg-id="",ТВ3
+http://91.220.135.165:81/15/index.m3u8
+#EXTINF:-1 tvg-id="",КИНОПРЕМЬЕРА HD
+http://45.159.74.13:80/Kinopremera/tracks-v1a1/mono.m3u8
```
Comment (Dum4G) [inline streams/ru.m3u:272]:
```suggestion
http://45.159.74.13:80/Kinopremera/index.m3u8
```

File: streams/ru.m3u [hunk 5]
```diff
@@ -248,3 +248,77 @@ http://stream0.tv41.ru/live.m3u8
...
+http://31.148.48.15:80/Dom_Kino_Premium_HD/index.m3u8
+#EXTINF:-1 tvg-id="",A1
+http://31.148.48.15:80/A1/index.m3u8
+#EXTINF:-1 tvg-id="",СТС Love
+http://31.148.48.15:80/STS_Love/index.m3u8
+#EXTINF:-1 tvg-id="",Sochi 24 HD
+http://serv30.vintera.tv:8081/sochi/sochi24_tv/playlist.m3u8
+#EXTINF:-1 tvg-id="",Симферополь-24 HD
+http://46.151.151.245:10001/play/a004/index.m3u8
+#EXTINF:-1 tvg-id="",Плюс ТВ BY
```
Comment (Dum4G) [inline streams/ru.m3u:293]: seems to be streams/by

File: streams/ru.m3u [hunk 6]
```diff
@@ -248,3 +248,77 @@ http://stream0.tv41.ru/live.m3u8
...
+#EXTINF:-1 tvg-id="",Остросюжетное HD
+http://213.172.73.133:8081/nimble_live/9009_%D0%9E%D1%81%D1%82%D1%80%D0%BE%D1%81%D1%8E%D0%B6%D0%B5%D1%82%D0%BD%D0%BE%D0%B5_HD/playlist.m3u8
+#EXTINF:-1 tvg-id="",Легендарный 24
+https://sevas.1tvcrimea.ru/legendarniy24.m3u8
+#EXTINF:-1 tvg-id="",Своё HD
+http://live.stv24.tv:8080/svoetv/HQ_video.m3u8
+#EXTINF:-1 tvg-id="",Star Family
+http://89.254.128.5:22280/star_family_hd/video.m3u8
+#EXTINF:-1 tvg-id="",Viju TV1000
+http://82.209.197.187:8080/udp/239.0.100.141:1234
```
Comment (Dum4G) [inline streams/ru.m3u:306]: That's not gonna stay for long

File: streams/ru.m3u [hunk 7]
```diff
@@ -248,3 +248,77 @@ http://stream0.tv41.ru/live.m3u8
...
+#EXTINF:-1 tvg-id="",Viju Русское кино
+http://82.209.197.187:8080/udp/239.0.100.82:1234
```
Comment (Dum4G) [inline streams/ru.m3u:308]: That's not gonna stay for long

File: streams/ru.m3u [hunk 8]
```diff
@@ -248,3 +248,77 @@ http://stream0.tv41.ru/live.m3u8
...
+#EXTINF:-1 tvg-id="",Феникс+Кино
+http://82.209.197.187:8080/udp/239.0.100.101:1234
```
Comment (Dum4G) [inline streams/ru.m3u:310]: That's not gonna stay for long

File: streams/ru.m3u [hunk 9]
```diff
@@ -248,3 +248,77 @@ http://stream0.tv41.ru/live.m3u8
...
+#EXTINF:-1 tvg-id="",6 ТВ горловка HD
+http://91.237.3.3:4022/udp/224.2.2.4:10000
```
Comment (Dum4G) [inline streams/ru.m3u:312]: That's not gonna stay for long

File: streams/ru.m3u [hunk 10]
```diff
@@ -248,3 +248,77 @@ http://stream0.tv41.ru/live.m3u8
...
+#EXTINF:-1 tvg-id="",Relax
+http://185.57.68.33:8091/232/tracks-v1a1/index.m3u8
+#EXTINF:-1 tvg-id="",Cinema Classic
+https://api.alpaca.t62a.com/hls/9114/stream0.m3u8
+#EXTINF:-1 tvg-id="",FAN
+http://94.230.240.17:8080/fan/index.m3u8
+#EXTINF:-1 tvg-id="",Viju TV1000
+http://109.171.45.109:81/udp/239.54.5.91:1234
```
Comment (Dum4G) [inline streams/ru.m3u:320]: That's not gonna stay for long

File: streams/ru.m3u [hunk 11]
```diff
@@ -248,3 +248,77 @@ http://stream0.tv41.ru/live.m3u8
...
+#EXTINF:-1 tvg-id="",Viju+ Serial
+http://94.229.250.73:8008/play/a003
+#EXTINF:-1 tvg-id="",Viju History
+http://109.171.45.109:81/udp/239.54.5.93:1234
```
Comment (Dum4G) [inline streams/ru.m3u:324]: That's not gonna stay for long

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Comment by freearhey** [comment]
Updated

**Review by Dum4G — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]
You may add : 

- Viju+ TV1000 Action : http://109.171.45.109:81/udp/239.54.5.111:1234
- Live Animal : http://109.171.45.109:81/udp/239.54.5.166:1224
- 8 Channel : http://109.171.45.109:81/udp/239.54.5.165:1224
- Pro Business : http://109.171.45.109:81/udp/239.54.5.167:1224

**Pull request merged**